src/components/Home/Hero/ExchangeWidget/ExchangeWidget.js

```diff
@@ -104,6 +115,18 @@ class ExchangeWidget extends Component {
     this.walletInputEl.focus();
   }
 
+  focusDestinationTag() {
```

Comment:
never used

Reply:
without these dest tag, paym id or memo is not passed to http request header. maybe am missing something